otterai/indexer.py

```diff
@@ -30,8 +30,20 @@ def get_file_type(file_path: str) -> Optional[str]:
         return 'frontend'
     elif ext in ['.sql', '.graphql']:
         return 'data'
-    elif ext in ['.test.js', '.test.ts', '.spec.py', '_test.go']:
+    elif ext in ['.test.js', '.test.ts', '.spec.py', '_test.go', '.spec.js', '.spec.ts']:
         return 'test'
+    elif ext in ['.env', '.env.*']:
```

Comment:
🔒 Ensure that handling of environment files is secure, especially if they contain sensitive information. Consider adding a comment to clarify this.